test/mocha.opts

```diff
@@ -1,4 +1,3 @@
---bail
```

Comment:
Can we not commit this change? I think it's useful if tests fail early in the CI.